### PR TITLE
Fix: APP-2551 - Crowdin Issues

### PR DESCRIPTION
**Pull request by sepehr2github**
## Description

- Fix crowdin labels on indicated places

Task: [APP-2551](https://aragonassociation.atlassian.net/browse/APP-2551)

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] I have selected the correct base branch.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [x] Any dependent changes have been merged and published in downstream modules.
- [x] I ran all tests with success and extended them if necessary.
- [ ] I have updated the `CHANGELOG.md` file in the root folder.
- [x] I have tested my code on the test network.

[APP-2551]: https://aragonassociation.atlassian.net/browse/APP-2551?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/containers/proposalStepper/index.tsx [hunk 1]
```diff
@@ -147,7 +154,10 @@ const ProposalStepper: React.FC<ProposalStepperType> = ({
           next();
         }}
       >
-        <SetupVotingForm pluginSettings={votingSettings} />
+        <SetupVotingForm
```
Comment (Fabricevladimir) [inline src/containers/proposalStepper/index.tsx]: Why not get the members inside `SetupMultisigVotingForm` instead and enable the query based on the `votingSettings` type? There's a typeguard called `isMultisigVotingSettings` so you won't even need to pass in additional props and can just use the `votingSettings` that we're already passing down.

File: src/containers/setupVotingForm/index.tsx [hunk 2]
```diff
@@ -13,12 +13,14 @@ import {
 } from 'utils/types';
 import SetupMultisigVotingForm from './multisig';
 import SetupTokenVotingForm from './tokenVoting';
+import {DaoMembersData} from 'hooks/useDaoMembers';
 
-type Props = {
+export type Props = {
   pluginSettings: SupportedVotingSettings;
+  daoMembers?: DaoMembersData;
```
Comment (Fabricevladimir) [inline src/containers/setupVotingForm/index.tsx]: See previous comment.

File: src/containers/setupVotingForm/multisig.tsx [hunk 3]
```diff
@@ -24,13 +24,18 @@ import {
...
+import {MultisigVotingSettings} from '@aragon/sdk-client';
 
 const MAX_DURATION_MILLS =
   MULTISIG_MAX_REC_DURATION_DAYS * MINS_IN_DAY * 60 * 1000;
 
 export type UtcInstance = 'first' | 'second';
 
-const SetupMultisigVotingForm: React.FC = () => {
+const SetupMultisigVotingForm: React.FC<SetupVotingProps> = ({
+  pluginSettings,
```
Comment (Fabricevladimir) [inline src/containers/setupVotingForm/multisig.tsx]: If you opt to pass the `pluginSettings` in here, can you type it accordingly so you don't have to cast it later on?

File: src/containers/smartContractComposer/emptyStateModal/emptyState.tsx [hunk 4]
```diff
@@ -35,7 +36,7 @@ const EmptyState: React.FC<Props> = props => {
           type="Object"
           object="smart_contract"
           title={t('scc.emptyState.title')}
-          description={t('scc.emptyState.description')}
+          description={htmlIn(t)('scc.emptyState.description')}
```
Comment (Fabricevladimir) [inline src/containers/smartContractComposer/emptyStateModal/emptyState.tsx:39]: Something is off about this: 
image

**Review by Fabricevladimir — CHANGES_REQUESTED** [review]
Thanks! Left a few comments.

**Comment by Fabricevladimir** [comment]
There's an issue with the gating as well for token voting: 
Screenshot 2023-10-24 065428

File: src/containers/setupVotingForm/index.tsx [hunk 5]
```diff
@@ -6,16 +6,13 @@ import {
...
-} from 'utils/types';
+import {ProposalFormData, StringIndexed} from 'utils/types';
 import SetupMultisigVotingForm from './multisig';
 import SetupTokenVotingForm from './tokenVoting';
+import {VotingSettings} from '@aragon/sdk-client';
 
-type Props = {
-  pluginSettings: SupportedVotingSettings;
+export type Props = {
+  pluginSettings: VotingSettings;
```
Comment (Fabricevladimir) [inline src/containers/setupVotingForm/index.tsx]: This type should be SupportedVotingSettings

File: src/locales/en/common.json [hunk 6]
```diff
@@ -475,8 +475,8 @@
       "buttonLabel": "Connect wallet now"
     },
     "gatingUsers": {
-      "tokenTitle": "Token not found",
-      "tokenDescription": "The wallet does not contain '{{tokenName}}' token. Please ensure that you have the necessary tokens in order to continue.",
+      "tokenTitle": "You can’t create a proposal",
+      "tokenDescription": "You don’t have the required voting power or token balance. Creating proposals in {{daoName}} requires at least {{amount}} ‘{{tokenSymbol}}’ in either voting power or token balance.",
```
Comment (Fabricevladimir) [inline src/locales/en/common.json:479]: This is being used to gate the tokenVoting proposal creation. You don't seem to be passing the actual values in.

**Review by Fabricevladimir — CHANGES_REQUESTED** [review]

File: src/containers/setupVotingForm/tokenVoting.tsx [hunk 7]
```diff
@@ -20,10 +20,10 @@ import {
 } from 'utils/date';
 import {DateTimeErrors} from './dateTimeErrors';
 import {ToggleCheckList, UtcInstance} from './multisig';
-import {ProposalFormData} from 'utils/types';
+import {ProposalFormData, SupportedVotingSettings} from 'utils/types';
 
 type Props = {
-  pluginSettings: VotingSettings;
+  pluginSettings: SupportedVotingSettings;
```
Comment (Fabricevladimir) [inline src/containers/setupVotingForm/tokenVoting.tsx]: This should be `VotingSettings/MajorityVotingSettings`, that's why there's a lint issue.

File: src/locales/en/common.json [hunk 8]
```diff
@@ -1299,9 +1305,9 @@
     },
     "dappConnect": {
       "title": "Select dApp",
-      "desc": "You can interact with the following dApps to add actions to your proposal. Learn more about this {{link}}.",
+      "desc": "You can interact with the following dApps to add actions to your proposal. Learn more about {{link}}.",
       "descLinkLabel": "in our guide",
-      "descLinkURL": "https://aragon.org/how-to/swap-tokens-with-cowswap",
+      "descLinkURL": "https://",
```
Comment (Fabricevladimir) [inline src/locales/en/common.json:1310]: This is weird. Are we sure wwe want this gone?

Comment (sepehr2github) [reply src/locales/en/common.json:1310]: It's not the only empty description list, which I don't know why

Comment (sepehr2github) [reply src/locales/en/common.json:1310]: Just checked the link, It's invalid! 404 indeed.

**Review by Fabricevladimir — APPROVED** [review]

**Pull request merged**